Automattic/vip-go-mu-plugins
# Add wp-parsely 2.6

## Description

This PR adds the version 2.6.0 of the Parsely plugin. This has been added by running:

```
bin/manage-wp-parsely-subtree.sh 2.6 2.6.0      
```

## Changelog Description

### Plugin Added: Parsely 2.6

We added Parse.ly 2.6.0 as an available option.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Set `wpvip_parsely_version` filter to return `2.6`
3. Check that Parsely works and it's reporting version 2.6.0
